shell/packages/sandstorm-db/db.js

```diff
@@ -112,6 +112,7 @@ DevApps = new Mongo.Collection("devapps");
 //
 // Each contains:
 //   _id:  The application ID string (as with Packages.appId).
+//   appId: The same as _id, included to mimic Packages.appId.
```

Comment:
Hmm, I think I would feel more comfortable if DevApps' _id became the package ID. Then all of the fields that exist in both have the same meaning in both, and we aren't repeating information in DevApps.